patterns/1-initial/innersource-ambassador.md

```diff
@@ -0,0 +1,73 @@
+## Title
+
+InnerSource Ambassadors
+
+## Patlet
+
+When driving InnerSource adoption through a large, decentralized organization it is hard to understand and address the local challenges that come up in different departments and regions.
+Local volunteers, called InnerSource Ambassadors, provide localized support by promoting InnerSource principles and acting as a communication bridge between their teams and the ISPO.
+
+## Problem
+
+The InnerSource Program Office (ISPO) cannot be everywhere at once within a large, decentralized organization. Without localized support, the ISPO struggles to understand the unique challenges and needs of different departments or regions and cannot effectively promote InnerSource practices across the organization.
+
+## Story
+
+In a multinational organization, the ISPO implemented InnerSource initiatives but noticed uneven adoption across departments. After appointing a local ambassador in one department, adoption rates improved as the ambassador tailored practices to the department's context and provided valuable feedback to the ISPO.
+
+## Context
+
+- The organization is large and has multiple independent departments.
+- An ISPO has been established to drive InnerSource adoption.
+- There is a need for consistent and tailored InnerSource implementation across diverse teams.
+- Employees with a passion for collaboration and InnerSource principles are present within the organization.
+
+## Forces
+
+- **Geographical and organizational scale:** The ISPO cannot physically or logistically engage with every department.
+- **Communication barriers:** Departments may have unique cultures and needs that are not immediately visible to the ISPO.
+- **Workload balance:** Ambassadors must fulfill this role alongside their primary responsibilities.
+- **Trust and credibility:** Ambassadors must be trusted by their teams and the ISPO.
+
+## Sketch (optional)
+
+*A diagram showing the ISPO at the center, with ambassadors positioned in various departments acting as bidirectional conduits of information and influence.*
+
+## Solutions
+
+- Identify and recruit volunteer ambassadors from across the organization who are enthusiastic about InnerSource. They support InnerSource goals while maintaining their primary organizational roles.
+- Train ambassadors on InnerSource principles, tools, and the goals of the ISPO.
+- Establish clear expectations for ambassadors, including acting as a liaison, promoting InnerSource practices, and providing feedback to the ISPO.
```

Comment:
This here might be the most valuable part of the Solution, maybe of the whole pattern.

Can we describe specific tasks that we expect the ambassador to perform?

Some ideas:

- consult local engineering managers when they are looking to make one of their projects ready for InnerSource
- help teams in the discovery phase when starting a major new implementation task - is there any existing project in the org that the team could reuse and contribute to, instead of building something entirely new? (this might require collaboration with other roles like Architects / Principals)